CHANGELOG.md

```diff
@@ -17,11 +17,54 @@ parameters as a HTTP POST.
 
 ## 0.21
 
+
+* Warning: Significant database updates when you switch from version 0.19 to 0.21 of Cromwell.
```

Comment:
I'd suggest rewording this to say that the database migration will run for a long time if there are large amounts of data already in the database.